returnn/tf/util/data.py

```diff
@@ -1014,9 +1031,10 @@ def copy_with_feature_last(self):
     assert self.feature_dim_axis is not None
     return self.copy_with_feature_dim_axis(-1)
 
-  def copy_add_batch_dim(self, batch_dim_axis):
+  def copy_add_batch_dim(self, batch_dim_axis, batch_dim=1):
```

Comment:
Remove the default value. We really should enforce this.
(Yes, that means, maybe some other code needs to be fixed. But that is what we should do then.)